packages/@aws-cdk/aws-stepfunctions-tasks/test/lambda/integ.invoke.qualifiers.ts

```diff
@@ -0,0 +1,80 @@
+import { Code, Function, Runtime } from '@aws-cdk/aws-lambda';
```

Comment:
This file is copied from `integ.invoke.ts` and adjusted to use a Version and an Alias.